api/src/services/cereNetworkService.js

```diff
@@ -1,15 +1,26 @@
 const { ApiPromise, WsProvider, Keyring } = require("@polkadot/api");
-const { config, CERE_MAINNET, CERE_MAINNET_WS_PROVIDER_URL_DEFAULT, MNEMONIC_EMPTY } = require("../constants/config");
-const {
-  NETWORKS,
-  CERE_WS_PROVIDER_URL
-} = process.env;
+const { cereTypes, blockchains } = require("../config");
+const { NETWORKS } = process.env;
 
 const networkParams = new Map();
 
 async function init() {
-  networkParams.set(CERE_MAINNET, await initNetwork(CERE_WS_PROVIDER_URL || CERE_MAINNET_WS_PROVIDER_URL_DEFAULT, MNEMONIC_EMPTY));
+  const promises = [];
+  const cere = blockchains[0];
+  cere.networks.forEach(network => {
+    const promise = async() => ({
+      name: network.name,
+      rpc: await initNetwork(network.rpcUrl, network.mnemonic)
```

Comment:
`mnemonic` is in NETWORKS that is taken from secrets. You should use it.

Reply:
Ok, I will create a  possibility to configure the mnemonic from the config file via env variables..

Reply:
Discussed. Mnemonic will be taken from NETOWRKS and task for migration created.

Reply:
resolved